go/proxyd/server.go

```diff
@@ -151,3 +192,12 @@ func instrumentedHdlr(h http.Handler) http.HandlerFunc {
 		respTimer.ObserveDuration()
 	}
 }
+
+func GetAuthCtx(ctx context.Context) string {
+	authUser, ok := ctx.Value(ContextKeyAuth).(string)
+	if !ok {
+		return "none"
```

Comment:
This assumes that nobody will use the name "none" which I think is a fair assumption, any reason to not use `""`

Reply:
See above note re: aliases - it lets us identify unauthenticated requests in the monitoring.

Reply:
I added some logic in `proxyd.go` that will prevent users from specifying `none` as the alias.